specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -28,16 +28,28 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-2022-03
+tag: package-2022-07
```

Comment:
You will need to add this package tag across all the readmes so that the SDK is generated for each language. You can look at the other readmes to see how the package tags are added